loadbalancer/fadein_test.go

```diff
@@ -92,8 +95,9 @@ func testFadeIn(
 			}
 		}()
 
+		// Split fade-in period into buckets and count how many times each endpoint was selected.
 		t.Log("test done", time.Now())
-		t.Log("CSV " + strings.Join(eps, ","))
+		t.Log("CSV timestamp," + strings.Join(eps, ","))
```

Comment:
here you likely need to change also:
[~/go/src/github.com/zalando/skipper]% ls skptesting/*fadein*
skptesting/analyze_fadein.r*  skptesting/run_fadein_test.sh*